tests/puppeteer/campaigns/functional/BO/shopParameters/general/maintenance/01_enableDisableShop.js

```diff
@@ -0,0 +1,134 @@
+require('module-alias/register');
+// Using chai
+const {expect} = require('chai');
+const helper = require('@utils/helpers');
+const loginCommon = require('@commonTests/loginBO');
+// Importing pages
+const BOBasePage = require('@pages/BO/BObasePage');
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const GeneralPage = require('@pages/BO/shopParameters/general');
+const MaintenancePage = require('@pages/BO/shopParameters/general/maintenance');
+const HomePage = require('@pages/FO/home');
+
+let browser;
+let page;
+const newMaintenanceText = 'Maintenance';
```

Comment:
This one too :sweat_smile:

Reply:
I use this one only in this test, then I think that it should be in this file not on the page.